DESCRIPTION

```diff
@@ -12,5 +12,5 @@ URL: https://github.com/eddelbuettel/ttdo/, https://dirk.eddelbuettel.com/code/t
 BugReports: https://github.com/eddelbuettel/ttdo/issues
 NeedsCompilation: no
 Encoding: UTF-8
-Imports: tinytest (>= 1.3.1), diffobj
+Imports: tinytest (>= 1.4.0), diffobj
```

Comment:
No I can only do that once that version exists (on CRAN).

Reply:
Also this is just ... sloppy, no?

![image](https://user-images.githubusercontent.com/673121/209970101-dd6b3bf3-7278-4685-9fcd-dd0edddf8408.png)

Reply:
This is also missing a default value.  Should it be `TRUE`?  Or `FALSE`?  Does it support `NULL` or `NA` or missingness?